.github/workflows/gh-pages-init.yml

```diff
@@ -27,10 +28,25 @@ jobs:
           mkdir -p ${{ env.FUELUP_DIR }}
           cp fuelup-init.sh ${{ env.FUELUP_DIR }}
 
-      - name: Deploy latest
+      - name: Deploy latest fuelup init
         uses: peaceiris/actions-gh-pages@v3
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           keep_files: true
           publish_dir: ${{ env.FUELUP_DIR }}
           destination_dir: ./
+
+      - name: Get latest tag
+        run: |
+          echo ${GITHUB_REF#refs/tags/} >> fuelup-version
+          cp fuelup-version ${{ env.FUELUP_VERSION_DIR }}
+        if: startsWith(github.ref, 'refs/tags')
```

Comment:
You don't need this because this job only runs on release publishing.

Reply:
good point